docs/modules/ROOT/pages/tutorials/tutorial.adoc

```diff
@@ -298,32 +306,50 @@ Let's start with the WordPress pod:
 TIP: In K9s you can open a shell in a pod by selecting that pod and then hitting the kbd:[S] key.
 
 [start=2]
-. Replace the files inside the pod with the ones from the restore:
+. Replace the files inside the pod with the ones from the restore. 
+  Using rsync here simplifies things by only copying files that are different from the target and also deleting any additional malicious files an attacker may have added:
 
 ....
-$ cd /var/www/html # or whatever value has `mountPath`
-$ mv data /
-$ rm -r *
-$ cp -R /data/wordpress-pvc/* .
+# apt update && apt install rsync
+# rsync -a --delete /restore/ /var/www/html/
```

Comment:
Why using the # here?

Also, one thing to note here is that root user has to run in the container in order to install rsync. On OpenShift, you cannot install additional packages since we're running with a non-privileged user. Maybe add a note about this?
Or maybe simply something like `rm -r /var/www/html/* && cp -r -p /restore/* /var/www/html/` is sufficient ? (not tested command) 
```suggestion
  Using rsync here simplifies things by only copying files that are different from the target and also deleting any additional malicious files an attacker may have added:
+
[source,bash]
----
apt update && apt install rsync
rsync -a --delete /restore/ /var/www/html/ # or replaced with rm/cp
----
```

Reply:
This gets me thinking... Why the rsync stuff anyways?

Wouldn't it be easier to simply mount the PVC with the restored data to the deployment? The method here could be used for more granular restore (f.e. only one file). But I think it's a bit overkill for a tutorial?

Reply:
It could work for this particular deployment. But if it were a statefulset (mariadb?), then it depends how the STS is configured. If it's a volume template, then you'd have to use cp/rsync, or recreate the PVC before starting up the STS again. Did I get this right?

Reply:
Yeah, but in this tutorial it's not a statefulset? I feel like this overcomplicates things in a tutorial that should just show "hey look it could look like this to use it".

Reply:
I had the `#` because I got a root user when exec'ing /bin/bash in the container. But this is all going to change after my new discovery that the restore *does* work, it just doesn't delete files that don't exist in the snapshot... So I will replace the whole rsync section with just a sentence mentioning this.

That means that in the case of a defaced WordPress site where an attacker has placed nasty things into `wp-content` to redistribute them, these would not be deleted by a restore. So it's not really a restore to a pristine state like the trick with `rsync --delete` from a separate PVC would be. But it's probably better/less complicated for a tutorial.

Reply:
No biggie :)

In a case of a nasty attack nuking the PVCs and hosts would be advisable anyways. You can't really be sure where the attacker placed backdoors and other nasty things.